environments/chaotic_pendulum.py

```diff
@@ -66,6 +66,12 @@ def get_default_radius_bounds(self):
         """
         return (0.5, 1.3)
 
+    def _get_default_ball_color(self):
```

Comment:
Feels like colors could be a protected member of the abstract class, then each child uses as many colors as it needs

Reply:
agreed